apps/ledger-live-mobile/src/navigation/DeeplinksProvider.tsx

```diff
@@ -333,11 +365,11 @@ const linkingOptions = {
 const getOnboardingLinkingOptions = (acceptedTermsOfUse: boolean) => ({
   ...linkingOptions,
   config: {
+    initialRouteName: NavigatorName.BaseOnboarding,
```

Comment:
Fix initial route name for deep links accessible in sync onboarding (Welcome screen instead of Portfolio screen)